### PR TITLE
chore: update issue templates

### LINKED ISSUE #1883: fix up issue forms/templates in samples
- [ ] add the option to go to the forum if you have a question
- [ ] convert issue templates into issue forms

### PR DESCRIPTION
**Pull request by rachel-fenichel**
Fixes https://github.com/google/blockly-samples/issues/1883

@cpcallen because of https://github.com/google/blockly/pull/7455

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
@BeksOmega because she might think it's funny

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    id: duplicates
+    attributes:
+      label: Check for duplicates
+      options:
+        - label: I have searched for similar issues before opening a new one.
+  - type: textarea
+    id: component
+    attributes:
+      label: Component
+      description: Which component has a bug?
```
Comment (maribethb) [inline .github/ISSUE_TEMPLATE/feature_request.yaml]: Which component does this request apply to?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Blockly Forum
+    url: https://groups.google.com/forum/#!forum/blockly
+    about: The Blockly developer forum, where you can ask and answer questions.
```
Comment (maribethb) [inline .github/ISSUE_TEMPLATE/config.yml]: Consider updating this to match christopher's wording

**Review by maribethb — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (rachel-fenichel) [reply .github/ISSUE_TEMPLATE/config.yml]: Yeah I'll match these to Christopher's and re-ping.

**Pull request merged**